src/GraphQLParser/Exceptions/GraphQLParserException.cs

```diff
@@ -8,6 +8,8 @@ namespace GraphQLParser.Exceptions;
 /// </summary>
 public class GraphQLParserException : Exception
 {
+    private static readonly char[] NewlineSeparator = ['\n'];
```

Comment:
Looks like this needs to be renamed to `_newlineSeparator` to match the naming rules enforced in this repo.

Reply:
And with the below suggestion this would be:

```suggestion
#if NETSTANDARD2_0
    private static readonly char[] _newlineSeparator = ['\n'];
#endif
```